app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -54,15 +54,19 @@ const AvailableActions = ({ course, current_user, updateCourse, courseCreationNo
   };
 
   const deleteCourseFunc = () => {
-    // The action is only available once a course has been removed from all campaigns.
-    if (course.published) {
```

Comment:
I think the easiest way to handle this would be to keep the alert but only if course.published AND Features.wikiEd.

Reply:
I think that due to this if statement: `if ((user.isAdvancedRole || user.admin) && (!course.published || !Features.wikiEd))` , when Features.wikiEd is true, the delete button is only rendered if the course is not published.  If the course is published, the delete button won't render at all, and therefore, deleteCourseFunc won’t be triggered. 

It worked previously because the button was rendered in P&E mode whether the course is published or not, so adding the ` if (course.published)` allowed the alert to show only if in P&E mode AND course.published.
 
But I also understand that we can keep the alert as a form of two-factor authentication for course deletion. Should I still keep it?

Reply:
We should keep the alert that requires the user to enter the title of the course, I think. But if there was an additional one that was just informational, that's not necessary.

Reply:
Yes🫡, the title confirmation prompt is still present (it's shown in the after screenshot  in the PR description). It was the additional one removed.